### ISSUE TITLE
Exercise 9 instructions incorrect

### ISSUE THREAD

**Issue by devnuhl** [php-school/learn-you-php#48]
The instructions say that the port will be the first argument to the program, however when verifying, the output errors make it seem like it is sending the port as a second parameter with the localhost ipv4 address as the first argument.

Upon making adjustments to test this, the program started working, but I have been unable to determine why it is expecting times 12 hours into the future. I could have understood if it was basing on GMT/UTC, but there is no mention of timezone we should be formatting for in the instructions.

**Comment by AydinHassan** [comment]
Thanks for reporting @devnuhl. You are right, the documentation is wrong, will get this fixed soon. As to the output, I'm not sure I follow, could you maybe show with some examples?

**Comment by devnuhl** [comment]
For some reason, when I ran it on my VM at work, the exercise verify script generated a datetime that was 12 hours ahead of what my datetime call generated. However, in trying to recreate it at home, I cannot. 

I've duplicated the code exactly as I had it earlier in the day, but without the same results. So it is possible that portion of my report is limited to the environment I was working in.

When I ran the verify script, it said the actual was 10:22:34 this morning, but it said that the expected  was 22:22:34.

**Comment by AydinHassan** [comment]
@devnuhl I've created a PR to fix the documentation - the incorrect date doesn't make much sense, we patch the code so it's all running under the same timezone, unless you manually set the time zone?

**Comment by devnuhl** [comment]
I will be trying to recreate it on the same machine today. Then I might be
able to track it down. Neither time value made much sense as I recall.

**Comment by devnuhl** [comment]
I had made a gist, but I finally realized what the issue was, and why it was so hard to find. I hadn't held shift long enough, so my format was wrong, and why it was 12h off when it tried to verify. That it was using a different timezone, so I couldn't compare it to the clock may not have made it easier to point out. 

It took rewriting the `$date` initialization as both a `date()` and `DateTime::format()` call to notice. That one was user error. Thanks for the quick fix on the documentation, though.
